src/main/java/com/redhat/documentation/asciidoc/extraction/model/Task.java

```diff
@@ -11,15 +11,17 @@
     private final Location location;
     private final PushableLocation pushableLocation;
     private final Map<String, Object> attributes;
+    private final String ignoreFile;
 
     public Task(Location location, PushableLocation pushableLocation) {
-        this(location, pushableLocation, Collections.emptyMap());
+        this(location, pushableLocation, Collections.emptyMap(), null);
```

Comment:
Pass an empty string instead. But with a list an empty collection would be better.